osu.Game/Overlays/Comments/DrawableComment.cs

```diff
@@ -24,23 +25,27 @@ public class DrawableComment : CompositeDrawable
         private const int margin = 10;
 
         public readonly BindableBool ShowDeleted = new BindableBool();
+        public readonly Bindable<CommentsSortCriteria> Sort = new Bindable<CommentsSortCriteria>();
 
         private readonly BindableBool childrenExpanded = new BindableBool(true);
+        private readonly Bindable<List<Comment>> childComments = new Bindable<List<Comment>>();
+        private readonly List<Comment> loadedChildren = new List<Comment>();
```

Comment:
How does `childComments` differ from `loadedChildren`?

Reply:
`ChildComments` is being modified on every `GetCommentReplies` request. However, the response of this request can include already loaded comments from the main request. so `losdedChildren` is used just to check, whether the comment already exists

Reply:
I'd either advocate for a naming change or an xmldoc here for clarity (ie. `requestedComments` vs `loadedComments`?)

Reply:
Removed by using `children.OldValue` in `childComments.BindValueChanged()`